src/django/api/exceptions.py

```diff
@@ -1,13 +1,19 @@
 from rest_framework.exceptions import APIException
 
 
-class ForbiddenException(APIException):
-    status_code = 403
-    default_detail = 'You are not allowed to perform this action.'
-    default_code = 'forbidden'
-
-
 class BadRequestException(APIException):
     status_code = 400
     default_detail = 'There was a problem with your request.'
     default_code = 'bad_request'
+
+
+class NotAuthorized(APIException):
+    status_code = 401
+    default_detail = 'You are not authorized to peform this action.'
+    default_code = 'not_authorized'
+
+
+class ForbiddenException(APIException):
```

Comment:
Consider using the built-in [PermissionDenied](https://www.django-rest-framework.org/api-guide/exceptions/#permissiondenied)

Reply:
9ff012610419ee9a2ca4e4e877c6c3eb726c7f30